### PR TITLE
Post DRAFT spec

### LINKED ISSUE #104: Complete spec
Draft at https://docs.google.com/document/d/1FrJ-trF1e4xONXPfa1zwlLWRfDTY0CUhxnw9FFK1jkk/edit?usp=sharing

### PR DESCRIPTION
**Pull request by jmertic**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207406625237354

### CODE REVIEW

File: specifications/cdsc-wg2-03.md [hunk 1]
```diff
@@ -1,486 +1,764 @@
...
+-   timeframe - *Enum* - One of day-ahead, hour-ahead, real-time
+
+##### Response Object
+
+-   id - *string* - (REQUIRED) - The id of the PowerSystemResource
+    associated with this price.
+
+-   unit - *string* - (REQUIRED) - Should be a currency code in
+    accordance with the [[ISO
+    4217]{.underline}](https://www.iso.org/iso-4217-currency-codes.html)
```
Comment (jmertic) [inline specifications/cdsc-wg2-03.md]: @jmertic to fix link

File: specifications/cdsc-wg2-03.md [hunk 2]
```diff
@@ -1,486 +1,764 @@
...
+
+-   [EECS Rules Fact Sheet 5: TYPES OF ENERGY INPUTS AND
+    TECHNOLOGIES](https://www.aib-net.org/sites/default/files/assets/eecs/facts-sheets/AIB-2019-EECSFS-05%20EECS%20Rules%20Fact%20Sheet%2005%20-%20Types%20of%20Energy%20Inputs%20and%20Technologies%20-%20Release%207.7%20v5.pdf)
+
+## 4. Terminology and Identification
+
+Within this standard, some entities SHALL only take specific values as
+outlined below.
+
+-   For fuel, the enum values SHALL be those in column 'Full code'
```
Comment (jmertic) [inline specifications/cdsc-wg2-03.md]: @jmertic to clean up formatting

File: specifications/cdsc-wg2-03.md [hunk 3]
```diff
@@ -1,486 +1,764 @@
...
-###	Normative references
+This specification offers a set of endpoints that contains the necessary
+information for carbon data calculations.
 
-The following documents are referred to in the text in such a way that some or all of their content constitutes requirements of this document. For dated references, only the edition cited applies. For undated references, the latest edition of the referenced document (including any amendments) applies.
+## 3. Normative references
 
-- IEC 61968/61970: Common Information Model 
-- ISO 4217: Currency Codes
+The following documents are referred to in the text in such a way that
```
Comment (jmertic) [inline specifications/cdsc-wg2-03.md:48]: @jmertic Add ISO date formatting

**Comment by Henry-WattTime** [comment]
Looks good for a comment draft.

**Comment by Pierre-VF** [comment]
Happy with the way it looks (given my comments on the call for the minor fixes) and to move it forward to a draft proposal by merging this PR.

File: specifications/cdsc-wg2-03.md [hunk 4]
```diff
@@ -1,913 +1,623 @@
...
+Attention is drawn to the possibility that some of the elements of this
+document may be the subject of patent rights. No party shall not be held
+responsible for identifying any or all such patent rights.
 
-Attention is drawn to the possibility that some of the elements of this document may be the subject of patent rights. No party shall not be held responsible for identifying any or all such patent rights.
+Any trade name used in this document is information given for the
+convenience of users and does not constitute an endorsement.
 
-Any trade name used in this document is information given for the convenience of users and does not constitute an endorsement.
+This document was prepared by the Power Systems Data Working Group of
```
Comment (k-stanford24) [inline specifications/cdsc-wg2-03.md]: Can we change this to "This draft deliverable was prepared by the LFESS’s Carbon Data Standards Consortium - Power System Data Working Group."?

**Comment by k-stanford24** [comment]
OK with moving forward as draft with the minor changes above.

**Pull request merged**